lib/apiservers/engine/backends/cache/repo_cache.go

```diff
@@ -128,8 +127,6 @@ func init() {
 // NewRespositoryCache will create a new repoCache or rehydrate
 // an existing repoCache from the portlayer k/v store
 func NewRepositoryCache(client *client.PortLayer) error {
-	defer trace.End(trace.Begin(""))
```

Comment:
I like you removed this relatively annoying debug traces!

Reply:
Yes, there were so many annoying useless debug traces clogging up the logs while I was debugging this issue.